lib/views/index.erb

```diff
@@ -1,30 +1,56 @@
 <% if @my_cart.count != 0 %>
-  <div id="shopping_cart">
-    Shopping cart
+<div class="cell">
```

Comment:
This is not a blocker, but about `if @my_cart.count != 0` 
I think that this kind of logic should NOT be present in the view. I would add a method to the moel instead. Something like this:
```ruby
def has_items?
  count != 0 # or self.count != 0 if you prefer
end
``` 
And in my view:
```ruby
<% if @my_cart.has_items? %>
```
You get a more readable code and separate the presentation aspects from the business logic. Does that make sense?